### PR TITLE
Remove the extended gamemode from Sage

### PR DESCRIPTION
**Pull request by Sinestia**
## About The Pull Request
( i hope you like my title ) 

I set extended's odds to 0, and moved it in the config with an explanation as to why. 

**I considered redistributing the points to normal secret_extended, but opted not to. If that's desirable just ask or submit it as a review and I'll be happy to do it.** 

## Why It's Good For The Game

Currently, extended running on Sage causes an issue where all 3 station objectives are blared out. This is caused by a goof I made when I fixed station objectives being disabled altogether on sage with the creation of NO_INTERCEPT_REPORT.

Aside from the bug, Sage still shouldn't be running normal extended.

It's not good when the gamemode can be meta'd by checking the cargo computer and 1/5th of the server immediately cryos. You can patch that up with admin intervention but frankly that seems very silly to me when secret_extended works perfectly fine. 

**That's why I opted to do this instead of submitting a fix first -- Because the gamemode is designed to give you 3 objectives and a fix won't change that.**
## Changelog
:cl:Froststahr
config: Disables "Extended" on Sage (Secret Extended is still enabled)
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
Some people do like the knowing extended mode to know when its safe to build

**Comment by ghost** [comment]
Some of the best rounds I've played were extended, the game is not just about antag rolling and combat. Extended allows for longer rounds where you can enjoy doing projects that just take too long for normal rounds.

Sage especially is for roleplay, disabling it for golden would make sense, but for sage I'd argue it's ideal.

**Comment by PowerfulBacon** [comment]
This doesnt remove extended, just the extended version that gives 3 station objectives.

**Comment by Sinestia** [comment]
i got someone with the clickbait oml

File: config/Sage/game_options.txt [hunk 1]
```diff
@@ -117,6 +116,9 @@ PROBABILITY METEOR 0
 PROBABILITY DEVIL 0
 PROBABILITY DEVIL_AGENTS 0
 
+## This gamemode is meta-able due to how it creates 3 station objectives, and is thus inappropriate for Sage. It also currently creates issues with NO_INTERCEPT_REPORT
+PROBABILITY EXTENDED 0
```
Comment (Crossedfall) [inline config/Sage/game_options.txt]:
```suggestion
## Creates issues when NO_INTERCEPT_REPORT is enabled
PROBABILITY EXTENDED 0 
```

Comment (Crossedfall) [reply config/Sage/game_options.txt]: This note should be mirrored to Golden's config as well.

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Mirror the note to Golden's config since that's the default one-shots use

**Review by ghost — DISMISSED** [review]

**Comment by Sinestia** [comment]
Mirrored

**Review by Crossedfall — APPROVED** [review]
Config applied

**Pull request merged**